### PR TITLE
Add new volunteer Aragorn

### PR DESCRIPTION
**Pull request by arbindmishra**

### CODE REVIEW

File: data/users/arbindmishra.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Aragorn",
+  "bio": "Web Developer",
+  "image": "https://avatars.githubusercontent.com/u/89216704?v=4",
```
Comment (RakeshSangem) [inline data/users/arbindmishra.json:4]: Original picture please:)

Comment (codewithvoid) [reply data/users/arbindmishra.json:4]: Avatar is fine @RakeshSangem :P

File: data/users/arbindmishra.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["javascript","Angular"]
```
Comment (codewithvoid) [inline data/users/arbindmishra.json:8]: AngularJS

Comment (arbindmishra) [reply data/users/arbindmishra.json]: Done

**Comment by codewithvoid** [comment]
Thank you @arbindmishra

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**